src/components/Dashboard/style.ts

```diff
@@ -27,3 +27,14 @@ export const UserCard = styled.div`
   padding: 0.25rem 1.5rem;
   border: 1px solid purple;
 `;
+
+export const UserNavigation = styled.button`
```

Comment:
```suggestion
export const UserNavigationButton = styled.button`
```

Talvez pra deixar mais claro na nomenclatura que se trata de um button?